charts/kubewarden-controller/values.yaml

```diff
@@ -32,3 +32,7 @@ policyServer:
   - apiGroup: "networking.k8s.io"
     resources:
     - ingresses
+
+tls:
+  useCertManager: true
```

Comment:
I agree with @flavio  we can remove useCertManager as it is mandatory and we don't provide any other choice